tests/integration_tests/security/test_spectre_meltdown.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# N.B.: Although this repository is released under the Apache-2.0, part of its test requires a
+# script from the third party "Spectre & Meltdown Checker" project. This script is under the
+# GPL-3.0-only license.
+"""Tests spectre / meltdown mitigation."""
+
+import pytest
+import requests
+
+from framework import utils
+from host_tools.network import SSHConnection
+
+
+CHECKER_URL = "https://meltdown.ovh"
+CHECKER_FILENAME = "spectre-meltdown-checker.sh"
+
+
+@pytest.fixture(name="spectre_meltdown_checker")
+def download_spectre_meltdown_checker(tmp_path):
+    """Download spectre / meltdown checker script."""
+    resp = requests.get(CHECKER_URL, timeout=5)
+    assert (
+        resp.ok
+    ), f"Failed to download the checker script. Status code: {resp.status_code}"
```

Comment:
Try `resp.raise_for_status()`, maybe it fits also

Reply:
Done. Thanks!